bridge_types/src/value.rs

```diff
@@ -0,0 +1,94 @@
+use std::fmt::{Display, Formatter};
```

Comment:
Since this is integral to Value and display_type() now should it just be merged into value.rs in vm?  Seems a bit fine grained pulling it out but if you feel strongly it should be in this crate I guess that is cool.  Vm is a terminal workspace currently and I think that is probably the correct move.

Reply:
weird. i thought b/c it was a proc macro it was going to yell at me but it's not an issue. moved.